File: SSIS/SSIS/Models/Core Models/Item.cs
using System;
using System.Collections.ObjectModel;
using System.ComponentModel.DataAnnotations;

namespace SSIS.Models
{
    public class Item
    {
        [Key]
        public String ItemCode { get; set; }
        public String Category { get; set; }
        public string Description { get; set; }
        public int ReorderLevel { get; set; }
        public int ReorderQuantity { get; set; }
        public int CurrentQuantity { get; set; }
        public string UnitOfMeasure { get; set; }
        public bool IsActive { get; set; }
        public Collection<ItemSupplier> ItemSuppliers { get; set; }

        public Collection<StockCard> StockCards { get; set; }
    }
}

File: Android/app/src/main/java/com/arun/api/Model/Item.java
package com.arun.api.Model;

import java.io.Serializable;

public class Item implements Serializable {
    String ItemCode;
    String Category;
    String Description;
    int ReorderLevel;
    int ReorderQuantity;
    int CurrentQuantity;
    String UnitOfMeasure;
    Boolean IsActive;

    public String getItemCode() {
        return ItemCode;
    }

    public void setItemCode(String itemCode) {
        ItemCode = itemCode;
    }

    public String getCategory() {
        return Category;
    }

    public void setCategory(String category) {
        Category = category;
    }

    public String getDescription() {
        return Description;
    }

    public void setDescription(String description) {
        Description = description;
    }

    public int getReorderLevel() {
        return ReorderLevel;
    }

    public void setReorderLevel(int reorderLevel) {
        ReorderLevel = reorderLevel;
    }

    public int getReorderQuantity() {
        return ReorderQuantity;
    }

    public void setReorderQuantity(int reorderQuantity) {
        ReorderQuantity = reorderQuantity;
    }

    public int getCurrentQuantity() {
        return CurrentQuantity;
    }

    public void setCurrentQuantity(int currentQuantity) {
        CurrentQuantity = currentQuantity;
    }

    public String getUnitOfMeasure() {
        return UnitOfMeasure;
    }

    public void setUnitOfMeasure(String unitOfMeasure) {
        UnitOfMeasure = unitOfMeasure;
    }

    public Boolean getActive() {
        return IsActive;
    }

    public void setActive(Boolean active) {
        IsActive = active;
    }
}
